common.mk

```diff
@@ -133,9 +133,8 @@ PRODUCT_PACKAGES += \
 PRODUCT_PACKAGES += \
     firmware_folders
 
-# Community APN list
-PRODUCT_COPY_FILES += \
-    $(COMMON_PATH)/rootdir/vendor/etc/apns-conf.xml:system/etc/apns-conf.xml
+
+USE_SODP_APNS ?= true
```

Comment:
Why is `USE_SODP_APNS ?= true` here and was `PRODUCT_COPY_FILES += \ ... ` moved below?

Just move them all below `-include device/sony/customization/customization.mk` for consistency, `?=` will only set it to a value if it wasn't assigned to before.